ossdbtoolsservice/driver/types/mysql_driver.py

```diff
@@ -229,7 +232,7 @@ def cursor(self, **kwargs):
         """
         self._conn.ping()
         # Create a new cursor from the current connection
-        cursor_instance = self._conn.cursor()
+        cursor_instance = self._conn.cursor(buffered=True)
```

Comment:
Can we keep buffered = False?
Do it improves the performance for large read queries?

Reply:
It does not improve perf, at best it is same